frontend/components/TeamSection/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { GradientText } from '../shared/Heading';
+import PersonCard from '../person-card';
+
+const TeamSection = ({ data }) => {
+  const {
+    title,
+    userOne,
+    userTwo,
+    userTitle,
+    description,
+    userOneImg,
+    userTwoImg
+  } = data;
+
+  const userOneData = {
+    image: userOneImg,
+    title: userOne,
+    subtitle: userTitle,
+    description: description
+  };
+  const userTwoData = {
+    image: userTwoImg,
+    title: userTwo,
+    subtitle: userTitle,
+    description: description
+  };
+
+  return (
+    <div className="text-center">
+      <GradientText
+        text={title}
+        gradientColor="green"
+        className="uppercase font-bold text-xxlg inline-block"
+      />
+      <div className="flex flex-col md:flex-row  justify-center mt-10 uppercase">
+        <PersonCard cardDetails={userOneData} rounded={true} boxShadow={true} />
```

Comment:
1) cards data should be an array of object and here is a map on cards data
2) A card should have a border and not a box shadow
3) A gap is required between cards

Reply:
could you please point to which shadow ? because its not clear in design.